shared/desktop/app/app-state.desktop.js

```diff
@@ -55,6 +56,7 @@ export default class AppState {
       height: 600,
       isFullScreen: null,
       isMaximized: null,
+      lastSentXLM: false,
```

Comment:
we shouldn't store this here. mobile and this should just use the config.json writing api in go

Reply:
https://github.com/keybase/client/pull/14318

Reply:
got it